workspace-server/src/main/java/gov/nasa/jpl/aerie/workspace/server/WorkspaceBindings.java

```
@@ -108,8 +127,17 @@ public void apply(final Javalin javalin) {
     });
     javalin.exception(NumberFormatException.class,
                       (ex, ctx) -> ctx.status(400).json(new FormattedError(ex)));
+    javalin.exception(SecurityException.class,
+                      (ex, ctx) -> ctx.status(500).json(new FormattedError(ex)));
+    javalin.exception(Exception.class, (ex, ctx) -> {
+      // Catch-all for unexpected issues
+      logger.error("Unexpected error processing workspace request", ex);
+      final var message = ex.getMessage() != null ? ex.getMessage() : "Unknown error.";
+      ctx.status(500).json(new FormattedError("UNKNOWN_ERROR", message, ex));
+    });
```

I did some testing on error cases and was able to hit this catch-all a couple times - good addition 👍 A few small recommendations:
* It's fine to not log noisy user problems like 404's, but we should add `logger.warn` or `logger.error` to handlers which indicate an "us problem", to help us debug what happened. Eg. at least `IOException` & `SQLException` currently return 500 but don't log.  Maybe `SecurityException` too since it either indicates a malicious user or a script gone wrong.
* Minor, but Javalin will sometimes throw specific HTTP exceptions which have already set HTTP status codes, and the catch-all handler will overwrite these as 500. I couldn't come up with a repro case without commenting out other handlers, but I think something like this would be a good defensive thing to put at the top of the catch-all handler:
```
  if (ex instanceof HttpResponseException hre) {
    ctx.status(hre.getStatus())
       .json(new FormattedError(hre));
    return;
  }
  ```

---

Added a commit to address this: [Log when a 500 is returned by the server](https://github.com/NASA-AMMOS/aerie/pull/1760/commits/bec83d3c5e1412062802a4aa29d82b539aaa8559)